index.md

```diff
@@ -38,7 +41,8 @@ Für die zweite Woche sind allerdings Grundkenntnisse zur Kommandozeile hilfreic
 Beide Teile des Kurses sind nicht nur für das Praktikum interessant,
 sondern auch für den weiteren Erfolg während und nach des Studiums.
 
-Alle im Kurs vorgestellten Programme sind [Free and Open-Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software){:target="_blank"}.
+Alle im Kurs vorgestellten Programme sind [Free and Open-Source Software](https://en.wikipedia.org/wiki/Free_and_open-source_software){:target="_blank"},
+bis auf [VSCode](/install/windows-10.html/#VSCode).
```

Comment:
Maybe add something along the lines of "bis auf VSCode (kostenlos, aber proprietär)."